tests/xammac_tests/CoreWlan/CWKeychainTests.cs

```diff
@@ -0,0 +1,85 @@
+//
```

Comment:
This file can go into the tests/monotouch-test, so that it's next to all the other tests.

The `xammac_tests` name will die eventually, and we'll only have 'monotouch-test` tests that will run on all platforms.